feedback-front/templates/layout.html

```diff
@@ -6,6 +6,10 @@
         <title>FeedBack Home</title>
         <link rel="apple-touch-icon-precomposed" href="images/apple-touch-icon.png">
         <link rel="stylesheet" href="{{ url_for('static', filename='css/uikit.min.css') }}">
+        <link rel="stylesheet" href="{{ url_for('static', filename='css/nv.d3.css') }}">
+        <script src="https://cdnjs.cloudflare.com/ajax/libs/d3/3.5.5/d3.min.js" charset="utf-8"></script> 
+        <script src="{{ url_for('static', filename='js/nv.d3.js') }}"></script>
```

Comment:
No - this should only be loaded in some files - not all files. Do not change layout.html at all